<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalSort</class>
 <widget class="QWidget" name="LocalSort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocalSort</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>pos与影像</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="posAndImage">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="pos">
          <attribute name="title">
           <string>POS</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>架次分类操作</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>选择要分架次的pos文件</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="selectPosFile">
                  <property name="text">
                   <string>打开</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>选择pos文件的类型</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="selectPosType"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QListWidget" name="posWidget"/>
              </item>
              <item>
               <widget class="QPushButton" name="startSort">
                <property name="text">
                 <string>开始架次分类</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>架次分类查看</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="checkBox">
                   <property name="text">
                    <string>架次全选</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="sortWidget" native="true"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="selectPath">
                   <property name="text">
                    <string>选择文件路径</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QProgressBar" name="progressBar">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QToolButton" name="startSave">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>34</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>开始传输</string>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cam1">
          <attribute name="title">
           <string>CAM1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="ImageSelect" name="widget1" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cam2">
          <attribute name="title">
           <string>CAM2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="ImageSelect" name="widget2" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cam3">
          <attribute name="title">
           <string>CAM3</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="ImageSelect" name="widget3" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cam4">
          <attribute name="title">
           <string>CAM4</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="ImageSelect" name="widget4" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cam5">
          <attribute name="title">
           <string>CAM5</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="ImageSelect" name="widget5" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cam6">
          <attribute name="title">
           <string>CAM6</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="ImageSelect" name="widget6" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageSelect</class>
   <extends>QWidget</extends>
   <header>imageselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
